static/app/utils/useOrganization.stories.tsx

```diff
@@ -1,10 +1,30 @@
+import {Fragment} from 'react';
+
 import StructuredEventData from 'sentry/components/structuredEventData';
 import storyBook from 'sentry/stories/storyBook';
 import useOrganization from 'sentry/utils/useOrganization';
 
 export default storyBook('useOrganization', story => {
-  story('useOrganization - via StructuredEventData', () => {
+  story('Default', () => {
     const org = useOrganization();
-    return <StructuredEventData data={org} />;
+
+    // Sort features and access alphabetically for the story use case
+    org.features.sort((a, b) => a.localeCompare(b));
+    org.access.sort((a, b) => a.localeCompare(b));
+
+    return (
+      <Fragment>
+        <p>useOrganization returns the currently selected organization.</p>
+        In the context of the Sentry dashboard, only a single organization cab be selected
+        at a time.
+        <p>
+          A common need to use <code>useOrganization</code> is to check for feature flags
+          via <code>organization.hasFeature(feature)</code>. Note that our feature names
```

Comment:
We should probably document the pattern that already exists 😅 

```suggestion
          via <code>organization.features.includes(feature)</code>. Note that our feature names
```

Reply:
Oh lol, my brain tricked me into thinking it exists